### PR TITLE
Add Batch Update and ClassCode developer guide

### PR DESCRIPTION
**Pull request by whoisjunhong**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -357,6 +358,23 @@ The implemented find by activity mechanism is facilitated by `FindByActivity`. I
 
 #### Enhancements
 
+The purpose of the batch update enhancement is update all students by `ClassCode` when the `Status` of a student in that `ClassCode` changes from `Negative` -> `Positive` and vice-versa.
+
+The batch update enhancement is facilitated by using `execute()` command in the `EditCommand` class.
+
+Batch update depends on the `Model` and `Person` class and methods to implement this enhancement.
```
Comment (lzf834) [inline docs/DeveloperGuide.md:365]: Details look good here. Clear, concise and relevant details.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -357,6 +358,23 @@ The implemented find by activity mechanism is facilitated by `FindByActivity`. I
...
+
+How the batch update works:
+
+* When `execute()` in `EditCommand` checks for a change in `Status` if the person to edit from `Negative` -> `Positive` and `Status` is not already `Positive`
+  * If true, a filtered `List` of students with the same `ClassCode`, students who are not `Positive`, not the current student being edited would be created.
+  * All students `Status` in the filtered `List` will be switched from `Negative` -> `Close-Contact`.
+* Conversely, when a student's `Status` changes from `Positive` -> `Negative`, `execute()` will check the current student being edited that there are no `Positive` statuses in `ClassCode`.
+  * If true, all students `Status` in the filtered `List`will be switched to `Negative`.
```
Comment (lzf834) [inline docs/DeveloperGuide.md:373]: Looks good here as well. Maybe we should discuss if we need a diagram here if it helps in making it clearer for the audience

**Review by lzf834 — APPROVED** [review]
Overall, good to go for our DG draft

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (whoisjunhong) [reply docs/DeveloperGuide.md:373]: I agree with this! Let's push for this during our next dg iteration 👍

**Pull request merged**